manifests/0000_51_olm_02_operator_clusterrole.yaml

```diff
@@ -219,6 +219,17 @@ rules:
     - get
     - list
     - watch
+  - apiGroups:
+    - olm.operatorframework.io
+    resources:
+    - clustercatalogs
+    verbs:
+    - get
+    - list
+    - watch
+    - create
+    - update
```

Comment:
If a server-side apply patch request results in creating or updating the resource do we need these permissions? I don't think we do, but I'll try to verify tomorrow when I can get another clusterbot cluster spun up

Reply:
Pretty sure `update` can be removed. I can't remember if `create` is required even if a `patch` operation performs the create.

Reply:
update removed in https://github.com/openshift/cluster-olm-operator/pull/73/commits/5b237d4e4d462f19ff7571661b839393bde6b676 - create had to stay.